core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -2956,6 +2956,22 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleAlterIsrRequest(request: RequestChannel.Request): Unit = {
+    val alterIsrRequest = request.body[AlterIsrRequest]
+
+    authorizeClusterOperation(request, CLUSTER_ACTION);
+
+    // TODO do we need throttling for this response?
```

Comment:
Probably reasonable to handle it the same way other inter-broker RPCs are handled.